Docs/src/tutorials/ogreFullSetupHandsOnGuide.md

```diff
@@ -0,0 +1,43 @@
+### Simple to follow instruction set for setup first project in self-build ogre1.12.5-project for beginners. (Linux only!)
+
+### Build and install ogre1-12-5:
+
+### Terminal-instructions:
+
+sudo apt-get install -y libgles2-mesa-dev libxt-dev libxaw7-dev  
+sudo apt-get install -y nvidia-cg-toolkit libsdl2-dev doxygen  
+
+cd replaceWIthCustomPathToYourWorkspace
+git clone --recursive --branch v1.12.5 https://github.com/OGRECave/ogre.git  
+mkdir build_me  
+cmake-gui  
+  * set absolute path to ogre directory and the build_me directory
+  * click on configure, then windows pops up “CMakeSetup“: only click on finish and wait until it‘s finished
+  * click again on configure and wait until it‘s finished
+  * click on generate and wait until it‘s finished
+
+cd build_me  
+cmake --build . --config release  
+make OgreDoc  
```

Comment:
this is certainly not needed for beginners

Reply:
why not? Then it can be installed.